NetKAN/KSP-Recall.netkan

```diff
@@ -0,0 +1,29 @@
+{
+    "spec_version"     : "v1.2",
+    "$kref"            : "#/ckan/spacedock/2434",
+    "$vref"            : "#/ckan/ksp-avc",
+    "identifier"       : "KSP-Recall",
+    "name"             : "KSP Recall",
+    "abstract"         : "Recall for KSP blunders, mistakes and borks.",
+    "license"          : [ "GPL-2.0", "restricted" ],
+    "release_status"   : "stable",
+    "ksp_version_min"  : "1.9.0",
+    "ksp_version_max"  : "1.9.1",
```

Comment:
```suggestion
```

These will be set automatically based on the version file, so they're not needed.